### ISSUE TITLE
little help

### ISSUE THREAD

**Issue by rikutatarin** [jlplenio/crude-twitch-viewer-bot#101]
Hi, I'm a bit new to this stuff.  So excuse my question.  Do the proxies we write in the Proxy List must contain a username and password?  The proxies I have consist of only IP and port.  When I use them, the software does not work.  Please someone help.

**Comment by jlplenio** [comment]
Should work with the `ip:port::` format. Are the proxies counted in the "Proxies Available"-Counter?
If your proxies are public, it is unlikely that they will work.

**Comment by rikutatarin** [comment]
yes when i go by «ip:port::», the counter works. but it says like: 
(1 dead,2dead,3dead,4dead...)

and yes,i use public proxies. imma buy private proxies. thanks for the reply. <3

**Comment by jlplenio** [comment]
Yes, public proxies cannot handle the load. Good luck.
